### ISSUE TITLE
hermit test openjdk failing - Zulu OpenJDK no source provided for darwin-amd64 arch

### ISSUE THREAD

**Issue by kirillzh** [cashapp/hermit-packages#138]
Running `hermit test openjdk` produces:
```
info:validate: Validating openjdk-11.0.8_10
info:validate: Validating openjdk-11.0.9_11
info:validate: Validating openjdk-11.0.10_9
info:validate: Validating openjdk-11.0.11_9-zulu11.48.21
info:validate: Validating openjdk-17.0.1_12
info:validate: Validating openjdk-17.0.2_8
info:validate: Validating openjdk@stable
info:validate: Validating openjdk@zulu
info:validate: Validating openjdk@latest
info:validate: Validating openjdk@11.0
info:validate: Validating openjdk@17
info:validate: Validating openjdk@17.0
info:validate: Validating openjdk@11
warn: openjdk: darwin-amd64: https://github.com/cashapp/hermit-packages.git/openjdk.hcl: openjdk-11.0.11_9-zulu11.48.21: no source provided
warn: openjdk: darwin-amd64: https://github.com/cashapp/hermit-packages.git/openjdk.hcl: openjdk-11.0.11_9-zulu11.48.21@zulu: no source provided
```

**Comment by alecthomas** [comment]
Should be fixed in 9e731b6

**Comment by kirillzh** [comment]
Fix https://github.com/cashapp/hermit-packages/commit/c844dc96b4baaecd83e3b5ddd8acc8c78cf4c256.
